.github/workflows/create-tag-workflow.yml

```diff
@@ -0,0 +1,44 @@
+name: Create Tag
+
+
+# This workflow reads the current version from setup.py and then creates a new tag and release named
+# for that version.
+# If a tag already exists with that name, the action will fail. Check log output of 'Create Release'
```

Comment:
If we want to avoid this, we could check for existing tag as per https://github.com/Crown-Commercial-Service/digitalmarketplace-utils/blob/0648a4290c4e99e9136084525f797d28111ce5db/.github/workflows/release.yml#L33-L39 ?